### PR TITLE
S04 us28 direction

### PR DESCRIPTION
**Pull request by jaldabaoth-code**

### CODE REVIEW

**Review by RaphaelBS-WCS — DISMISSED** [review]
Approved

File: templates/leaflet/index.html.twig [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<div class="example-wrapper">
+    <h1>Hello {{ controller_name }}! ✅</h1>
+
+    This friendly message is coming from:
+    <ul>
+        <li>Your controller at <code><a href="{{ '/home/zurabi/Documents/WildCS/Projet3/orleans-202103-php-project-jobpermut/src/Controller/LeafletController.php'|file_link(0) }}">src/Controller/LeafletController.php</a></code></li>
+        <li>Your template at <code><a href="{{ '/home/zurabi/Documents/WildCS/Projet3/orleans-202103-php-project-jobpermut/templates/leaflet/index.html.twig'|file_link(0) }}">templates/leaflet/index.html.twig</a></code></li>
+    </ul>
+</div>
+{% endblock %}
```
Comment (LucyMerlier) [inline templates/leaflet/index.html.twig]: this template is useless

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

File: templates/leaflet/index.html.twig (hunk 1 above)
Comment (jaldabaoth-code) [reply templates/leaflet/index.html.twig]: yes you are right, i didn't looked there was tamplate.

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**